lib/oggvorbis/lib/x64/libvorbis.la

```diff
@@ -0,0 +1,41 @@
+# libvorbis.la - a libtool library file
```

Comment:
Why have you added this file? We don't use libtool so this file should not be necessary.